### PR TITLE
feat: Implement responsive footer component

### PR DESCRIPTION
**Pull request by SydneyTechnologies**

### CODE REVIEW

**Comment by SydneyTechnologies** [comment]
@Harithmetic1 

Switch to the footer branch and `npm run dev` and checkout the responsiveness of the footer. Check if it meets the standards from the design and if not let me know what I need to change

**Comment by Harithmetic1** [comment]
1. The Contact Us and Follow Us containers should have a 
`justify-content: center;`
`gap: 160px;` 
on desktop view only
2. Instead of creating a div of height 0.1px, give the container a 
`border-top: 1px solid <color>`
`border-bottom: 1px solid <color>`
The divs are not visible on the Safari browser because they are too small; a border would suffice for all browsers.

File: src/components/shared/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+          <ul className="mb-10 text-xs text-[#FFFFFF] font-extralight md:flex gap-5 ">
+            <li className="mt-4">EXPERIENCES</li>
+            <li className="mt-4">CONTACT</li>
+          </ul>
+        </div>
+
+        <div className="w-full h-[0.1px] bg-white bg-opacity-20" />
+
+        <div className="flex justify-evenly items-center text-sm ">
```
Comment (Harithmetic1) [inline src/components/shared/Footer.jsx]: use a border on this container instead of the divs above and below

Comment (Harithmetic1) [inline src/components/shared/Footer.jsx]: On desktop give it a justify-center and put a gap

**Comment by Harithmetic1** [comment]
Looks good overall. Make the changes, and I'll merge

File: src/components/shared/Footer.jsx (hunk 1 above)
Comment (SydneyTechnologies) [reply src/components/shared/Footer.jsx]: @Harithmetic1 

Change has been made to the footer,  a couple of things might not be exactly the same. I think there is some issue with the font. But we can review that later I think It looks good at the moment.

**Pull request merged**